src/components/Dashboard/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './dashboard.scss';
+import Card from './DashboardCard';
+
+class Dashboard extends React.Component {
+  // Use case Example for dashboard card
+  // Card component takes children and props of height width and title
+  // Height and width are required and take a string of expected height and width respectively as shown below
+  // Title is optional and defaults to empty string if not provided
+  render(){
+    return(
+        <Card height="40vh" width="1060px" title="Engagement Trends">
```

Comment:
You can omit the closing card tag and replace it with something like this
`<Card height="40vh" width="1060px" title="Engagement Trends" />`